app/src/Command/SendAuthMailRequestCommand.php

```diff
@@ -230,11 +237,13 @@ private function createAuthEmail(
         Msgs $message,
         string $mailFrom,
         ?string $fromName,
-        array $body
+        array $body,
+        ?User $user
```

Comment:
Petit détail (ça facilitera la relecture du diff les prochaines fois) :


```suggestion
        ?User $user,
```